manager/testdata/new-blueprints/scenario4/arrow-flight-config-module.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     values: {}
   capabilities:
     - capability: read
-      scope: cluster
+      scope: workload
```

Comment:
@revit13 This config module is configuring a cluster level service.   That's why it's scope was cluster.  It does of course pass data to that service that is for a particular workload. @froesef @roee88 Which do you think it should be?

Reply:
After rethinking that I delete this change...  IIUC the config module is deployed per cluster.

Reply:
As I understand it, cluster scope means that something is deployed once per cluster, i.e. cross-workloads. This is not our case.

Reply:
@shlomitk1 It's not now.  The question is should it be?  If we have 15k workloads do we really need 15k of these config modules?

Reply:
@simanadler We need to configure the service, and the configuration differs from one workload to another. It is not a "global" service that can be accessed by the workload.